### DISCUSSION TITLE
create path with a slash value in a route

### DISCUSSION THREAD

**Discussion by mrsibs** [apioo/fusio#537] [Q&A] [answered]
Hi, i have a sql query that queries data with a slash value like this:
select name,address from tableA where id = '2022/H/ABC'

i refer to the manual to create a route and the manual state that "This route matches to any value except a slash".
Please advise how can i create a route with a path that have a slash value.

Thank you and im really appreaciate your prompt response.

**Comment by chriskapp** [comment, marked as answer]
Hi @mrsibs, so you can use an path like `/foo/*bar` and bar will then contains all values including a forward slash.

**Reply by mrsibs** [reply]
i have tested and success..thank you very much!
